@here/harp-mapview/lib/MapView.ts

```diff
@@ -1490,7 +1490,28 @@ export class MapView extends THREE.EventDispatcher {
     }
 
     /**
-     * Moves the camera to specified geocoordinates, sets desired zoom level and adjusts yaw and
+     * The method that sets the camera to the desired angle (`pitch`) and distance (in meters) to
+     * the `target` location.
+     *
+     * @param target The location to look at.
+     * @param distance The distance of the camera to the target in meters.
+     * @param pitch The camera tilt angle in degrees.
+     * @param yaw The camera yaw angle in degrees.
+     */
+    lookAt(target: GeoCoordinates, distance: number, pitch: number = 0, yaw: number = 0) {
+        const zoomLevel = MapViewUtils.calculateZoomLevelFromDistance(distance, this);
+        const geoLocation = MapViewUtils.getCameraCoordinatesFromTargetCoordinates(
+            target,
+            zoomLevel,
+            yaw,
+            pitch,
+            this
+        );
+        this.setCameraGeolocationAndZoom(geoLocation, zoomLevel, yaw, pitch);
```

Comment:
Can we not set the (projected)camera position directly instead of setCameraGeolocationAndZoom?

Reply:
Yes! Thanks! It solved my problem.